docs/user-manual/onboarding-guide/programguide

```diff
@@ -0,0 +1,112 @@
+
+- [ ] **Decide who will be the [CiviForm Admin](https://docs.civiform.us/overview/how-does-civiform-work\#civiform-admin-experience)** across your civic entity  
+      - [ ] Decide who will own and maintain your CiviForm instance in the long term (IT team, Digital Services or other). This will be the person who creates the Forms and Programs on CiviForm.
+
+- [ ] **Review and evaluate** the [Recommended CiviForm Launch Team](https://docs.google.com/document/d/1AacsuDbMmVfmBuWWSNRz3nOUYR3WG5\_KDnDgxrc6z0A/edit)  
+      - [ ] Consider if any new staffing or resources are needed for the implementation / maintenance of CiviForm and/or the programs.
+
+- [ ] **Choose programs to put in CiviForm:** Determine which programs will benefit most from using CiviForm (e.g., benefit applications, complaint forms, event registrations, permit applications).  
+      - [ ] Evaluate programs most likely to benefit from CiviForm by using these resources:  
+            - [ ]  [CiviForm Fellowship - Program Assessment](https://docs.google.com/document/d/10TTk68evp-X8iJ2W3S2--MpiqMeg1Uf0jTysbZITEdg/edit?usp=sharing)  
+            - [ ]  [CiviForm - Program Evaluation Worksheet ](https://docs.google.com/spreadsheets/d/1\_NNMln-LOAxxXCoYgh1oI0uaS6YhoyMoBvudaiKRMm4/edit?resourcekey=0-KPl3Y1YgUYty81OKHbRMpQ\#gid=629433048)  
+      - [ ] Examine selected programs to identify opportunities to streamline overlapping questions:  
+            - [ ] Review the information gathered on the application for each program and remove any unnecessary questions.  
+            - [ ] Work with program stakeholders including program administrators to understand the administrative and policy requirements for different programs.   
+            - [ ] Identify common questions across programs (Ex. Name, Address, income).  
+            - [ ] Explore ways to consolidate these questions to streamline reuse.  
+            - [ ] Consider creating a “Common Intake” form as a “Get Started”.  
+      - [ ] For each program:  
+            - [ ] Decide on who the program leader/owner and primary point of contact will be.  
+            - [ ] Decide how applications to the program will be reviewed. Will they be reviewed in CiviForm, or pulled into an external system? If an external system, identify the technical PoCs.
+
+- [ ] **Join the CiviForm community**  
+      - [ ] Understand how the CiviForm community works.  
+            - [ ] [Community member responsibilities](https://docs.civiform.us/governance-and-management/governance/roles-committees-and-responsibilities)  
+            - [ ] [Governance Processes](https://docs.civiform.us/governance-and-management/governance/governance-processes)  
+      - [ ] Learn about  the [CiviForm community's communication channels](https://docs.civiform.us/governance-and-management/governance/communication).  
+            - [ ] Join [CiviForm Slack](https://civiform.slack.com/) and introduce yourself in the \#introductions channel.  
```

Comment:
**question:** Does the \ add formatting?

Reply:
This md was mostly outputted from Google Docs. I will have to check what the '\#' is supposed to do.

Reply:
The backslash escapes generates literal characters. In this case, it will generate a # instead of it being a comment.